### DISCUSSION TITLE
Libplist Issue

### DISCUSSION THREAD

**Discussion by the-guitar-man** [mikebrady/shairport-sync#1639] [General]
I am trying to build with Airplay 2, but I am running into a library issue with libplist-dev. I have it installed, but the version installed is 1.12. To complete the Shairport build, I get an error saying I need 2.2.0 or later. However, when I run sudo apt update, the package does not update. Similarly, if I run sudo apt install libplist-dev, it says it is already at the latest version.

I am guessing this is because I am trying to do this on a Raspberry Pi 3 running Raspbian 9 (Stretch). Maybe with this older OS, I cannot install the latest libplist-dev package? Any help appreciated, thanks.

**Comment by mikebrady** [comment]
Thanks for the question. I am afraid that it really needs 2.2.0 or later. Perhaps you could build and install it from source, but TBH it might be easier to update to the latest-and-greatest unless you have other constraints.
